src/core/issues-notes.js

```diff
@@ -118,7 +118,37 @@ function handleIssues($ins, ghIssues, conf) {
           }
         }
         $tit.find("span").text(text);
-        if (report.title) {
+        if (report.title && githubAPI) {
+          var labelListSpan = document.createElement('span');
+          Array.from(ghIssue.labels)
+            .forEach((label, index) => {
```

Comment:
This “forEach” creates things (labels), so it’s a ` .map(createLabel)`.  And those labels get reduce()’d into the outter container.

Reply:
> This “forEach” creates things (labels), so it’s a .map(createLabel). And those labels get reduce()’d into the outer container.

Thanks for this, looks great to implement.